.github/workflows/postgres.yaml

```diff
@@ -29,9 +29,7 @@ jobs:
       - name: Install Requirements
         run: pip install -r requirements.txt
       - name: Tests
-        run: |
-          python -m pytest tests/db
```

Comment:
Nit: You still have `tests/db` in your `.dockerignore` file.

Not related to this file, just didn't know where else to put this comment.

Reply:
Fixed.